app/views/badges/show.slim

```diff
@@ -2,34 +2,25 @@
   #badge-name
     h1
       = @badge.name
-  #badge-logo
-    |  <img src="
-    = @badge.logo_url
-    | " alt="
-    = @badge.name
-    | " id="badge-logo-img"> 
-    .col-md-12
-      = @badge.description
-    .col-md-12
-      = content_tag  :div, "", id: "share-button", data: { description:"Que badge maravilhosa, já viste?", image: "http://intra.seium.org" + @badge.logo_url}
-  #badge-users
+  #badge-logo.col-md-4
+    = image_tag @badge.logo_url, id:"badge-logo-img",  alt:@badge.name
     br
-    - if (users = @badge.users).size > 0
+    = @badge.description
+  #badge-users.col-md-8[style="margin-left: 20px"]
```

Comment:
Could this inline style tag be moved to one of the existing .scss files? If it is needed at all :)